main.go

```diff
@@ -37,5 +38,7 @@ func main() {
 		cmd.CmdAdmin,
 	}
 	app.Flags = append(app.Flags, []cli.Flag{}...)
-	app.Run(os.Args)
+	err := app.Run(os.Args)
```

Comment:
@lunny `cmd/serve.go` always returns `nil`, so this isn't really an issue here :)